src/protections/WordList.ts

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```

Comment:
Unless you're actually assigning copyright to the foundation, I suspect this should be your name. For example:

`Copyright 2020 Emi Tatsuo`